gensim/models/utils_any2vec.py

```diff
@@ -3,7 +3,6 @@
 #
 # Author: Shiva Manne <s.manne@rare-technologies.com>
 # Copyright (C) 2018 RaRe Technologies s.r.o.
-
```

Comment:
-1: why this change? There definitely should be a blank line.

Reply:
no, see http://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_numpy.html